### PR TITLE
Add new sa03 questions

### PR DESCRIPTION
**Pull request by kelmryan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1233,6 +1243,7 @@ We are so thankful for every contribution, which makes sure we can deliver top-n
 
 **[⬆ Back to Top](#table-of-contents)**
 
+### To serve Web traffic for a popular product your chief financial officer and IT director have purchased 10 ml large heavy utilization Reserved Instances (RIs) evenly spread across two Availability Zones: Route 53 is used to deliver the traffic to an Elastic Load Balancer (ELB). After several months, the product grows even more popular and you need additional capacity As a result, your company purchases two C3.2xlarge medium utilization RIs You register the two c3 2xlarge instances with your ELB and quickly find that the ml large instances are at 100% of capacity and the c3 2xlarge instances have significant capacity that's unused Which option is the most cost effective and uses EC2 capacity most effectively?
```
Comment (danieldanielecki) [inline README.md:1246]: duplicated, please delete

File: README.md [hunk 2]
```diff
@@ -116,9 +123,12 @@ We are so thankful for every contribution, which makes sure we can deliver top-n
...
 | 6   | [Which of the following are valid statements about Amazon S3? (Choose 2 answers)](#which-of-the-following-are-valid-statements-about-amazon-s3-choose-2-answers)
 | 7   | [How can the domain's zone apex, for example, 'myzoneapexdomain.com', be pointed towards an Elastic Load Balancer?](#how-can-the-domains-zone-apex-for-example-myzoneapexdomaincom-be-pointed-towards-an-elastic-load-balancer)
+| 7   | [How can the domain's zone apex, for example, 'myzoneapexdomain.com', be pointed towards an Elastic Load Balancer?](#how-can-the-domains-zone-apex-for-example-myzoneapexdomaincom-be-pointed-towards-an-elastic-load-balancer)
```
Comment (danieldanielecki) [inline README.md:131]: duplicated, please delete

File: README.md [hunk 3]
```diff
@@ -116,9 +123,12 @@ We are so thankful for every contribution, which makes sure we can deliver top-n
...
+| 4   | [Your website is serving on-demand training videos to your workforce. Videos are uploaded monthly in high resolution MP4 format. Your workforce is distributed globally often on the move and using company-provided tablets that require the HTTP Live Streaming (HLS) protocol to watch a video. Your company has no video transcoding expertise and it required you may need to pay for a consultant. How do you implement the most cost-efficient architecture without compromising high availability and quality of video delivery'?](#your-website-is-serving-on-demand-training-videos-to-your-workforce-videos-are-uploaded-monthly-in-high-resolution-mp4-format-your-workforce-is-distributed-globally-often-on-the-move-and-using-company-provided-tablets-that-require-the-http-live-streaming-hls-protocol-to-watch-a-video-your-company-has-no-video-transcoding-expertise-and-it-required-you-may-need-to-pay-for-a-consultant-how-do-you-implement-the-most-cost-efficient-architecture-without-compromising-high-availability-and-quality-of-video-delivery)
```
Comment (danieldanielecki) [inline README.md:127]: duplicated, please delete

File: README.md [hunk 4]
```diff
@@ -18,6 +20,7 @@
 
 1. Always happy to answer your questions on Udemy's Q&A's and outside :)
 2. Failed? Please submit a screenshot of your exam result and request a refund (via our upcoming platform, not possible on Udemy); we'll always accept it.
+2. Failed? Please submit a screenshot of your exam result and request a refund (via our upcoming platform, not possible on Udemy); we'll always accept it.
```
Comment (danieldanielecki) [inline README.md:23]: duplicated, please delete

File: README.md [hunk 5]
```diff
@@ -79,6 +82,10 @@
 
 - Improve multiple questions, typos, and fix broken links.
 
+**[v1.1.0](../../releases/tag/v1.1.0): May 8, 2024.**
+
```
Comment (danieldanielecki) [inline README.md:86]: duplicated, please delete

File: README.md [hunk 6]
```diff
@@ -6,10 +6,12 @@
 
 ❣️ Please support us by purchasing this course on Udemy in an interactive version with the [discounted link](https://www.udemy.com/course/aws-certified-solutions-architect-associate-saa-c03-exams-v/?referralCode=CEA49DDC465C505A263A). If you're working for a company, you could most probably easily claim this expense during preparation for your exam. For us, it's to be, or not to be, in the game.
 
+🛍️ Alternatively, you can buy the PDF with those questions on [Etsy](https://ditectrev.etsy.com/listing/1529635334) or [Google Play Books](https://play.google.com/store/books/details?id=Z6MGEQAAQBAJ).
```
Comment (danieldanielecki) [inline README.md:9]: duplicated, please delete

File: README.md [hunk 7]
```diff
@@ -6,10 +6,12 @@
...
 🛍️ Alternatively, you can buy the PDF with those questions on [Etsy](https://ditectrev.etsy.com/listing/1529635334) or [Google Play Books](https://play.google.com/store/books/details?id=Z6MGEQAAQBAJ).
 
 ## ✨ This course is unlike any Amazon Web Services Certified (AWS Certified) Solutions Architect Associate (SAA-C03) course you will find online.
 
+✋ Join a live online community and a course taught by industry experts and pass the Amazon Web Services Certified (AWS Certified) Solutions Architect Associate (SAA-C03) confidently. We aim to build an ecosystem of Information Technology (IT) certifications and online courses in cooperation with the technology industry. We believe it will give our students 100% confidence in the pacing market in an open-source environment. We are just at the beginning of our way, so it's even better for you to join now!
```
Comment (danieldanielecki) [inline README.md:14]: duplicated, please delete

File: Test.md
Comment (danieldanielecki) [inline Test.md:1]: It looks like a copy/paste of README.md?

File: README.md (hunk 1 above)
Comment (kelmryan) [reply README.md:1246]: It does not appear in my repo or your current Readme. Where wlse are you seeing the duplicate?

File: README.md (hunk 2 above)
Comment (kelmryan) [reply README.md:131]: I don't think this is a duplicate. Might need to run a dos2unix as maybe there is a char diff?

File: README.md (hunk 3 above)
Comment (kelmryan) [reply README.md:127]: Same as above. This isn't a duplicated in my version

File: README.md [hunk 8]
```diff
@@ -116,9 +123,12 @@ We are so thankful for every contribution, which makes sure we can deliver top-n
...
+| 3   | [A user has launched an EC2 instance. The instance got terminated as soon as it was launched. Which of the below mentioned options is not a possible reason for this?](#a-user-has-launched-an-ec2-instance-the-instance-got-terminated-as-soon-as-it-was-launched-which-of-the-below-mentioned-options-is-not-a-possible-reason-for-this)
```
[the remark this answers is not in the document]
Comment (kelmryan) [reply README.md:126]: See above

File: README.md (hunk 1 above)
Comment (kelmryan) [reply README.md:1246]: I was on the wrong branch Will remove

File: Test.md
Comment (kelmryan) [reply Test.md:1]: Correct only with removed - [x] so I could test myself

Comment (danieldanielecki) [reply Test.md:1]: We also sell such options: https://www.etsy.com/shop/Ditectrev?section_id=43955909
but if you can do it by yourself, of course it's your gain ;-)

**Comment by kelmryan** [comment]
Let me split off my branch for the test.md and re push

**Comment by danieldanielecki** [comment]
I'll fix duplicates in `README.md` directly on `main`...

**Comment by danieldanielecki** [comment]
Duplicates fixed in https://github.com/Ditectrev/AWS-Certified-Solutions-Architect-Associate-SAA-C03-Practice-Tests-Exams-Questions-Answers/commit/3f933660ae2083455ec94c3ec3f3416befea180d.

**Pull request merged**